src/shims/intrinsics/mod.rs

```diff
@@ -393,47 +393,10 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
         F: Float + Into<Scalar<Provenance>>,
     {
```

Comment:
This is not worth its own function any more (and having 2 functions with the same name is anyway potentially confusing).

Reply:
Inlined the function